src/SprykerEco/Zed/Braintree/Business/Payment/Transaction/Handler/CaptureItemsTransactionHandler.php

```diff
@@ -7,16 +7,17 @@
 
 namespace SprykerEco\Zed\Braintree\Business\Payment\Transaction\Handler;
 
+use Generated\Shared\Transfer\BraintreeTransactionResponseTransfer;
 use Generated\Shared\Transfer\TransactionMetaTransfer;
 
-class CaptureTransactionHandler extends AbstractTransactionHandler implements CaptureTransactionHandlerInterface
+class CaptureItemsTransactionHandler extends AbstractTransactionHandler implements CaptureItemsTransactionHandlerInterface
```

Comment:
Well, we should not use Handler in the class name.
But if it's legacy code, I guess we can leave it as it is for it
https://spryker.atlassian.net/wiki/spaces/RFC/pages/364479366/ACCEPTED+RFC+Disallow+Handler+Helper+Manager+models

Reply:
We have a lot of handlers here, but I'm not sure I have to change so many classes because they've existed since 2017